test/fw/bin/pbs_snapshot

```diff
@@ -0,0 +1,193 @@
+    sys.excepthook = trap_exceptions
+
+sys.excepthook = trap_exceptions
+
+
+def usage():
+    msg = """
+Usage: pbs_snapshot -o <path to output tar file> [OPTION]
+
+    Take snapshot of a PBS system and optionally capture logs for diagnostics
+
+    -d <pbs_diag>                     diag directory to use as input
+    -H <hostname>                     hostname to operate on
+                                      Defaults to value of PBS_SERVER
+    -l <loglevel>                     set log level to one of INFO, INFOCLI,
+                                      INFOCLI2, DEBUG, DEBUG2, WARNING, ERROR
+                                      or FATAL
+    --service-logs=<num days>         number of service logs to collect
+    --accounting-logs=<num days>      number of accounting logs to collect
+    --additional_hosts=<hostname>     collect data from additional hosts
+                                      'hostname' is a comma separated list
+    --map=<file>                      file to store the map of obfuscated data
+    --obfuscate                       obfuscates sensitive data
+    --version                         print version number and exit
+"""
+
+    print msg
+
+
+if __name__ == '__main__':
+
+    # Arguments to PBSSnapUtils
+    out_dir = None
+    diag_path = None
+    server_host = None
+    log_level = "INFOCLI2"
+    acct_logs = 30  # Capture 30 days of accounting logs by default
+    srvc_logs = 5   # Capture 5 days of service logs by default
+    additional_hosts = None
+    map_file = None
+    anonymize = False
+    create_tar = True
+
+    log_file = "pbs_snapshot.log"
+
+    PtlConfig()
+
+    # Parse the options provided to pbs_snapshot
+    try:
+        sopt = "d:H:l:o:h"
+        lopt = ["accounting-logs=", "service-logs=",
+                "additional-hosts=", "map=", "obfuscate", "version"]
+        opts, args = getopt.getopt(sys.argv[1:], sopt, lopt)
+    except:
+        usage()
+        sys.exit(1)
+
+    for o, val in opts:
+        if o == "-d":
+            diag_path = CliUtils.expand_abs_path(val)
+        elif o == "-o":
+            out_dir = val
+        elif o == "-H":
+            server_host = val
+        elif o == "-l":
+            log_level = val
+        elif o == "-h":
+            usage()
+            sys.exit(0)
+        elif o == "--accounting-logs":
+            acct_logs = int(val)
+        elif o == "--service-logs":
+            srvc_logs = int(val)
+        elif o == "--additional-hosts":
+            additional_hosts = val
+        elif o == "--map":
+            map_file = val
+        elif o == "--obfuscate":
+            anonymize = True
+        elif o == "--version":
+            print ptl.__version__
+            sys.exit(0)
+        else:
+            sys.stderr.write("Unrecognized option")
+            usage()
+            sys.exit(1)
+
+    fmt = '%(asctime)-15s %(levelname)-8s %(message)s'
+    level_int = CliUtils.get_logging_level(log_level)
```

Comment:
I'm actually not very familiar with providing logging configuration via an INI file, but I'm not sure that I see the value in providing that to the users. We already have a log level argument separately. As far as the format of the logs are concerned, at least for pbs_snapshot, I don't think it matters as the users won't really care about the generated logs until pbs_snapshot errors out, at which point they will send the logs to us and we already have in place the format that we prefer.

Do you still think that it's needed? I would rather not add it just because other commands have it.